sdk/schemaregistry/schema-registry-avro/README.md

```diff
@@ -30,8 +27,9 @@ npm install @azure/schema-registry-avro
 ### SchemaRegistryAvroSerializer
 
 Provides API to serialize to and deserialize from Avro Binary Encoding plus a
-header with schema ID. Uses [SchemaRegistryClient](../schema-registry) to get
-schema IDs from schema content or vice versa.
+header with schema ID. Uses
+[SchemaRegistryClient](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/schemaregistry/schema-registry)
```

Comment:
in the future we should make this a link to the reference docs

Reply:
Added note to #10950